src/Init/Data/Int/LemmasAux.lean

```diff
@@ -151,4 +151,46 @@ theorem bmod_bmod_of_dvd {a : Int} {n m : Nat} (hnm : n ∣ m) :
   obtain ⟨k, rfl⟩ := hnm
   simp [Int.mul_assoc]
 
+theorem bmod_eq_of_le_of_lt {x : Int} {y : Nat} (hle : -y ≤ x * 2) (hlt : x * 2 < y) :
+    x.bmod y = x := by
```

Comment:
You can replace the proof with `bmod_eq_self_of_le (by omega) (by omega)`, and perhaps it would be a good idea to change one of the names to make the difference more clear.

Reply:
done! now it's called `bmod_eq_self_of_le_mul_two`